src/Codeception/Util/PathResolver.php

```diff
@@ -124,4 +124,13 @@ private static function isWindows($dirSep = DIRECTORY_SEPARATOR)
     {
         return ($dirSep == '\\');
     }
+
+    public static function isPathAbsolute($path)
+    {
+        if (DIRECTORY_SEPARATOR === '/') {
+            return mb_substr($path, 0, 1) === DIRECTORY_SEPARATOR;
```

Comment:
I think that `substr($path, 0, 1)` would work fine  because there is no other utf-8 character which starts with the same byte as '/'. 
All other places where `mb` functions are called specify `utf-8` as charset parameter.